### PR TITLE
Improve sso error feedback

### PR DESCRIPTION
**Pull request by MKodde**
1. The RedirectToFeedbackPageExceptionListener catches the InvalidRequestMethodException and InvalidBindingException
2. The original exception message is levered into the error page as it  is very descriptive.
3. Finally the error message is rendered with a translated title (the  exception message is always displayed in English.).

This PR was created on top of #641 
See task 2 of: https://www.pivotaltracker.com/story/show/164121560

### CODE REVIEW

File: src/OpenConext/EngineBlockBundle/EventListener/RedirectToFeedbackPageExceptionListener.php [hunk 1]
```diff
@@ -159,6 +161,10 @@ public function onKernelException(GetResponseForExceptionEvent $event)
         } elseif ($exception instanceof StuckInAuthenticationLoopException) {
             $message         = 'Stuck in authentication loop';
             $redirectToRoute = 'authentication_feedback_stuck_in_authentication_loop';
+        } elseif ($exception instanceof InvalidRequestMethodException | $exception instanceof InvalidBindingException) {
```
Comment (pablothedude) [inline src/OpenConext/EngineBlockBundle/EventListener/RedirectToFeedbackPageExceptionListener.php]: Did you deliberately use a bitwise operator?

File: src/OpenConext/EngineBlockBundle/Controller/FeedbackController.php [hunk 2]
```diff
@@ -365,6 +365,25 @@ public function stuckInAuthenticationLoopAction()
...
 
+    /**
+     * @return Response
+     */
+    public function noAuthenticationRequestReceivedAction(Request $request)
+    {
+        // The exception message is used on the error page. As mostly developers or other tech savy people will see this
+        // message. The ExceptionListener is responsible for setting the message on the feedback_custom field.
+        $session = $request->getSession();
+        $message = $session->get('feedback_custom');
```
Comment (pablothedude) [inline src/OpenConext/EngineBlockBundle/Controller/FeedbackController.php]: Could we safely assume that the 'feedback_custom' parameter is always from the requestId we would expect?

File: src/OpenConext/EngineBlockBundle/EventListener/RedirectToFeedbackPageExceptionListener.php (hunk 1 above)
Comment (MKodde) [reply src/OpenConext/EngineBlockBundle/EventListener/RedirectToFeedbackPageExceptionListener.php]: No, I did not intend using that! Well spotted :ok_hand:

File: src/OpenConext/EngineBlockBundle/Controller/FeedbackController.php (hunk 2 above)
Comment (MKodde) [reply src/OpenConext/EngineBlockBundle/Controller/FeedbackController.php]: When used correctly, yes we can assume this message is set. But you are right. We cannot assume correct usage. I'm adding a safeguard.

**Pull request merged**